### PR TITLE
Firefox on windows icon not showing properly

### PR DESCRIPTION
**Pull request by filipmanole**
The solution I found was to Increase the font size to separate the dot from 'i'.

### CODE REVIEW

**Comment by filipmanole** [comment]
On Windows, the letter 'i' is not centred perfectly as on Mac.

File: src/components/TokenSelect/TokenSelect.styles.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ export const MaxButtonStyle = css`
   align-self: center;
   border-radius: 0.125rem;
   font-weight: 600;
-  font-size: 0.75rem;
+  font-size: 0.81rem;
```
Comment (piersss) [inline src/components/TokenSelect/TokenSelect.styles.tsx:76]: https://github.com/airswap/airswap-web/blob/develop/CONTRIBUTING.md#styled-components

> We define dimensions using rem based on 16px

**Comment by filipmanole** [comment]
The font is not supported on windows. The solution was to change it to a web-safe font.

**Review by piersss — APPROVED** [review]
LGTM

**Pull request merged**